Assignments/2019-07-18/LinkedList/src/Coding/Training/List.java

```
@@ -0,0 +1,90 @@
+package Coding.Training;
+
+public class List implements LinkedList {
+    private Node head;
+    private Node current;
+    private int length;
+
+    public List() {
+        head = current = new Node(null, null);
+        length = 0;
+    }
+
+    // Return the item at index i
+    public Object get(int i) {
+        current = head.next;
+        int j = 0;
+
+        while (current != null && j < i) {
+            current = current.next;
+            j++;
+        }
+
+        return current.value;
```

It's ok for now. But what if I give an `i` larger than the length? We should throw an exception if we give a out-of-bound index.

---

Got it! I am gonna fix this problem in next mission.